chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/DeveloperManager.java

```diff
@@ -288,7 +298,15 @@ public DeveloperDTO create(DeveloperDTO dto)
 
         DeveloperDTO created = developerDao.create(dto);
         dto.setId(created.getId());
-        createOrUpdateTransparencyMappings(dto);
+
```

Comment:
Since this code is the basically same as the code on line 197 and in the same class there could easilly be a method for both instead - but, it's just a suggestion not a requirement